packages/stream-tile/src/tile-document.ts

```diff
@@ -307,7 +293,7 @@ export class TileDocument<T = Record<string, any>> extends Stream {
       prev: this.tip,
       id: this.state$.id.cid,
     }
-    const commit = await _signDagJWS(this.api, rawCommit)
+    const commit = await TileDocument._signDagJWS(this.api, rawCommit)
```

Comment:
why make it a static method?

Reply:
To indicate that it is a helper function and to mimic the prior behavior outside the class, but can be invoked now as part of the object during the test-case without the need of importing it separately (hence the move inside the class).